src/network.zig

```diff
@@ -884,24 +884,49 @@ pub const Protocols = struct {
 	pub const blockUpdate = struct {
 		pub const id: u8 = 7;
 		pub const asynchronous = false;
+
+		pub const BlockUpdate = struct {
```

Comment:
How about reusing `mesh_storage.BlockUpdate` for this? Then you could also adjust the interface of `renderer.mesh_storage.updateBlock` to take a `BlockUpdate`